h/static/scripts/polyfills.js

```diff
@@ -0,0 +1,15 @@
+// ES2015 polyfills
+require('core-js/es6/promise');
+require('core-js/fn/object/assign');
+
+// URL constructor, required by IE 10+,
+// Microsoft Edge.
+window.URL = require('js-polyfills/url').URL;
```

Comment:
This is also a somewhat more brutal approach than [the polyfill itself takes](https://github.com/inexorabletash/polyfill/blob/15de2d3e7f21abd20304940a19d2436c80290e7f/url.js#L22). Can we not use the feature detection built into the polyfill here?